src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATE_LINE = System.lineSeparator();
+    private static final String SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String readFile = readFile(fromFileName);
+        String preparedData = processData(readFile);
+        writeToFile(preparedData, toFileName);
+    }
+
+    public String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(SEPARATE_LINE);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read file", e);
```

Comment:
Make it more informative (print file name). Check all your exception messages

Reply:
It is not resolved, please check ALL your exception messages